github3/pulls.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, dest, direction):
         if dest.get('repo'):
             self._repo_name = dest['repo'].get('name')
             self._repo_owner = dest['repo']['owner'].get('login')
+            self.repository = Repository(self.repository, self)
```

Comment:
Hey Guys, I made both these fixes and my addition now passes MOST of the tests; its fails the ones that are labeled flake-8.